piikii-output-cache/redis/src/main/kotlin/com/piikii/output/redis/config/RedisConfig.kt

```diff
@@ -72,6 +75,21 @@ class RedisConfig {
             .cacheDefaults(redisCacheConfiguration)
             .build()
     }
+
+    @Bean
+    fun redissonClient(redisProperties: RedisProperties): RedissonClient? {
+        val config = Config()
+        val address = REDISSON_HOST_PREFIX + redisProperties.host + ":" + redisProperties.port
```

Comment:
Bean이라 어차피 실행 중 한번 수행이 보장되므로 상수를 따로 빼지 않아도 될 것 같아요.
(호불호이긴한데, 해당 클래스 자체의 책임 상 상수로 빼지 않아도 될 것 같아서요)

코틀린 문자열 템플릿으로 작성하면 아래와 같이 해볼 수 있을 것 같아요.
```suggestion
        val address = "rediss://$redisProperties.host:$redisProperties.port"
```

Reply:
핫 제 자바 습관이 나왔네요🥹 깔끔한 코드 감사해용 :)